From 7b280422607073e3a43e95a48d528bb9a7a1f8c9 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Fri, 24 Aug 2018 10:50:43 +0800
Subject: [PATCH 4486/5242] MLK-19323 ARM64: dts: imx8qxp: change back usdhc
 clock parent to PLL0

commit  d2c6d8eb75e0380eb61d29e83100ef8f66327a6a from
https://source.codeaurora.org/external/imx/linux-imx.git

According to IC reply, 8QXP B0 still has some issue on its DPLL.
Though we still not find any issue when usdhc use PLL0(DPLL), but better
to change back to PLL1(APLL) just in case any problem. So this patch change
back the usdhc clock parent to PLL1.

To track the history, refer to commit 7834eee6dfa8 ("MLK-17188-2
ARM64: dts: imx8qxp: assign usdhc clock parent") and commit 0611d9138e6e
("MLK-18003 ARM64: dts: imx8qxp: change back usdhc clock parent to PLL0").

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
index c13f563..f98f1fa 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
@@ -2996,7 +2996,7 @@
 			<&clk IMX8QXP_CLK_DUMMY>;
 		clock-names = "ipg", "per", "ahb";
 		assigned-clocks = <&clk IMX8QXP_SDHC0_SEL>, <&clk IMX8QXP_SDHC0_DIV>;
-		assigned-clock-parents = <&clk IMX8QXP_CONN_PLL0_CLK>;
+		assigned-clock-parents = <&clk IMX8QXP_CONN_PLL1_CLK>;
 		assigned-clock-rates = <0>, <400000000>;
 		power-domains = <&pd_conn_sdch0>;
 		fsl,tuning-start-tap = <20>;
@@ -3014,7 +3014,7 @@
 			<&clk IMX8QXP_CLK_DUMMY>;
 		clock-names = "ipg", "per", "ahb";
 		assigned-clocks = <&clk IMX8QXP_SDHC1_SEL>, <&clk IMX8QXP_SDHC1_DIV>;
-		assigned-clock-parents = <&clk IMX8QXP_CONN_PLL0_CLK>;
+		assigned-clock-parents = <&clk IMX8QXP_CONN_PLL1_CLK>;
 		assigned-clock-rates = <0>, <200000000>;
 		power-domains = <&pd_conn_sdch1>;
 		fsl,tuning-start-tap = <20>;
@@ -3032,7 +3032,7 @@
 			<&clk IMX8QXP_CLK_DUMMY>;
 		clock-names = "ipg", "per", "ahb";
 		assigned-clocks = <&clk IMX8QXP_SDHC2_SEL>, <&clk IMX8QXP_SDHC2_DIV>;
-		assigned-clock-parents = <&clk IMX8QXP_CONN_PLL0_CLK>;
+		assigned-clock-parents = <&clk IMX8QXP_CONN_PLL1_CLK>;
 		assigned-clock-rates = <0>, <200000000>;
 		power-domains = <&pd_conn_sdch2>;
 		status = "disabled";
-- 
1.7.9.5

